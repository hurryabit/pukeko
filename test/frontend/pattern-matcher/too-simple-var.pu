f :: a -> a
f x =
  match x with
  | y -> y
