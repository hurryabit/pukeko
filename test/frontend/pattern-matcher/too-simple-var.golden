test/frontend/pattern-matcher/too-simple-var.pu:5:5:
unexpected 'y'
expecting data constructor

